src/Mailgun/Mailgun.php

```diff
@@ -67,8 +67,10 @@ class Mailgun
      * @param string              $apiEndpoint
      * @param Hydrator|null       $hydrator
      * @param RequestBuilder|null $requestBuilder
+     *
+     * @internal Use Mailgun::configure or Mailgun::create instead.
      */
-    public function __construct(
+    public /* private */ function __construct(
```

Comment:
I did not change the constructor. This PR is only a doc fix. 

I added this comment to show that it should be treated as "private". This not code. It is a way to show that the constructor is private without breaking BC.